pkg/chunk/chunk.go

```diff
@@ -184,7 +184,14 @@ var writerPool = sync.Pool{
 // Encode writes the chunk into a buffer, and calculates the checksum.
 func (c *Chunk) Encode() error {
 	var buf bytes.Buffer
+	return c.EncodeTo(&buf)
```

Comment:
Can we just do `return c.EncodeTo(nil)`?

Reply:
yup definitively.